### PR TITLE
Fix the issue that ExceptionDetails class doesn't decorate the EventData attribute correctly

### PR DESCRIPTION
**Pull request by yantang-msft**
As titled. Without this fix, it may throw exception when track exception
@pharring @SergeyKanzhelev @lmolkova

### CODE REVIEW

File: src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs [hunk 1]
```diff
@@ -77,5 +77,30 @@ protected ExceptionDetails(string fullName, string name)
             stack = "";
             parsedStack = new List<StackFrame>();
         }
+
+        /// <summary>
+        /// Creates a new instance of ExceptionDetails from a System.Exception and a parent ExceptionDetails.
+        /// </summary>
+        internal static ExceptionDetails CreateWithoutStackInfo(Exception exception, ExceptionDetails parentExceptionDetails)
```
Comment (SergeyKanzhelev) [inline src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs]: this is generated file. Please do not modify manually

Comment (SergeyKanzhelev) [inline src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs]: It's patial so just move implementation into the different file

Comment (dnduffy) [inline src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs]: Yes, any changes here will be lost the next time we regenerate.

Comment (yantang-msft) [inline src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs]: Moved the implementation to a third partial class to keep the pattern consistent.

File: src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs [hunk 2]
```diff
@@ -29,45 +29,45 @@ namespace Microsoft.ApplicationInsights.Extensibility.Implementation.External
     using System.Collections.Concurrent;
     using System.Collections.Generic;
 
-    
-    
+
```
Comment (pharring) [inline src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs]: I would undo all changes in this file. Even though there are only whitespace changes, it's still different from what was auto-generated by the tool and could cause problems.

Comment (yantang-msft) [inline src/Core/Managed/Shared/Extensibility/Implementation/External/ExceptionDetails_types.cs]: Reverted. VS auto formatted it for me when I copied the source code. :)

**Comment by pharring** [comment]
👍

**Comment by SergeyKanzhelev** [comment]
Should it be in `2.4.0-beta1` or can wait?

+ @cijothomas

**Pull request merged**